spec/support/capybara_helpers.rb

```diff
@@ -21,92 +21,99 @@ def expect_css(css)
     expect(page).to have_css(css)
   end
 
-  # Select 2 helpers
+  def click_add_nested
+    find("a.has_many_add").click
+  end
 
-  def select2_options_container
-    find(".select2-container")
+  def on_nested_ctx(resource_number, &block)
+    within("li.has_many_container fieldset:nth-child(#{resource_number + 1}) ") do
+      block.call
+    end
   end
 
-  def select2_input
-    find(".select2-search__field")
+  def expect_slimselect_options_count_to_eq(count, id=nil)
+    id = slimselect_original_select_id unless id
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Use the double pipe equals operator ||= instead.